packaging/RPMS/Fedora/rabbitmq-server.spec

```diff
@@ -188,7 +190,9 @@ fi
 %attr(0755, rabbitmq, rabbitmq) %dir %{_localstatedir}/lib/rabbitmq
 %attr(0750, rabbitmq, rabbitmq) %dir %{_localstatedir}/lib/rabbitmq/mnesia
 %attr(0755, rabbitmq, rabbitmq) %dir %{_localstatedir}/log/rabbitmq
-%dir %{_sysconfdir}/rabbitmq
+%attr(2755, -, -) %dir %{_sysconfdir}/rabbitmq
```

Comment:
Changed [RPM](https://github.com/rabbitmq/rabbitmq-server-release/pull/30/files#diff-6d40e15a00ae78ad694619969f3cfffbR192)  and [Debian](https://github.com/rabbitmq/rabbitmq-server-release/pull/30/files#diff-2ffafa0624636e2c09c5d972bd285483R36)